README.md

```diff
@@ -289,6 +289,15 @@ Timeout can be passed to Devnet's HTTP server. This makes it easier to deploy an
 cargo run -- --timeout TIMEOUT
 ```
 
+### State archive mode
+
+With state archive capacity set to `full` devnet will store full state history. The default mode is `none`, where no old states are stored.
```

Comment:
Needs the comma, and perhaps the project name can be capitalized:

```suggestion
With state archive capacity set to `full`, Devnet will store full state history. The default mode is `none`, where no old states are stored.
```

Reply:
Fixed in c727b6cd25e1aaa1debbc6ded48ac22872545c48